diff --git a/ipv8/deprecated/community.py b/ipv8/deprecated/community.py
index c4ae4c6..e004428 100644
--- a/ipv8/deprecated/community.py
+++ b/ipv8/deprecated/community.py
@@ -43,7 +43,7 @@ _DNS_ADDRESSES = [
 ]
 
 
-BOOTSTRAP_TIMEOUT = 30.0 # Timeout before we bootstrap again (bootstrap kills performance)
+BOOTSTRAP_TIMEOUT = 2.0 # Timeout before we bootstrap again (bootstrap kills performance)
 
 
 class PacketDecodingError(RuntimeError):
