diff --git a/twisted/plugins/ipv8_plugin.py b/twisted/plugins/ipv8_plugin.py
index f567569..a1bf5ef 100644
--- a/twisted/plugins/ipv8_plugin.py
+++ b/twisted/plugins/ipv8_plugin.py
@@ -44,7 +44,11 @@ class IPV8ServiceMaker(object):
         """
         Main method to startup IPv8.
         """
-        self.ipv8 = IPv8(get_default_configuration())
+        config = get_default_configuration()
+        config['overlays'] = [o for o in config['overlays'] if
+                              o['class'] not in ('DHTDiscoveryCommunity', 'HiddenTunnelCommunity',
+                                                 'TrustChainCommunity')]
+        self.ipv8 = IPv8(config)
 
         def signal_handler(sig, _):
             msg("Received shut down signal %s" % sig)
